modules/claims_api/README.md

```diff
@@ -25,7 +25,7 @@ To remove documentation for both environments, use the default `document: false`
 The implementation of `production: false` required additional changes to the [rswag_override](https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/rswag_override.rb) file. Due to this, the [rswag](https://github.com/rswag/rswag) repo should be checked periodically to update the `example_group_finished` method if needed.
 
 ## Seeds
-Run `rake veteran:load_sample_vso_data ` to load Veteran organizations and a Veteran representative.
+Run `rake veteran:load_sample_vso_data ` to load Veteran organizations and a Veteran representative
```

Comment:
Was this change intentional?

Reply:
Yes, it was to trigger the tests to re-run in jenkins, they seemed to have been stuck and this inconsequential change triggered them to run again.

Reply:
Oh I see. I would not consider that a valid reason to make a change in the repository. Can you please change it back?

In the future, you can re-run Jenkins jobs by clicking 'Details' on one of the checks in the CI pipeline, and then select 'Re-Run all jobs' on the CI check page.

Reply:
Yep, I clicked Details, and Re-Run jobs multiple times and still it got stuck, so I tried this approach and it worked.  I just reversed this change.  Thanks.